source/Backend/BackendInit.js

```diff
@@ -199,6 +336,7 @@ function getMonthlyGoals(monthStr) {
  * user is not signed in (i.e bypassing the authentication).
  */
 function getUserID() {
+    // source: https://github.com/firebase/firebase-js-sdk/issues/462#:~:text=you%20can%20easily%20implement%20that%20on%20your%20own%20with%20a%20couple%20of%20lines%3A
```

Comment:
What is this link for?

Reply:
It's there as an explanation of where I got the getUserID() code from. I can remove it if you'd like